plugin/src/App/init.lua

```diff
@@ -97,7 +156,16 @@ function App:init()
 	end
 end
 
-function App:addNotification(text: string, timeout: number?, actions: { [string]: {text: string, style: string, layoutOrder: number, onClick: (any) -> ()} }?)
+function App:didUnmount()
```

Comment:
```suggestion
function App:willUnmount()
```

Reply:
I'm curious what the motivation is here, why the name change suggestion?

Reply:
`didUnmount` is not a valid Roact method. It should be `willUnmount`.

Reply:
Whoops!